main.go

```diff
@@ -452,8 +452,14 @@ func cmdNew(c *cli.Context) error {
 			return scanner.Err()
 		}
 		title = scanner.Text()
+		if title == "" {
+			file = time.Now().Format("2006-01-02") + ".md"
+			title = time.Now().Format("2006-01-02")
```

Comment:
I suggest to fix like below.

```
title = time.Now().Format("2006-01-02")
file = title + ".md"
```
Because first Now() may not be second Now().